md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessTemplateDAO.java

```diff
@@ -345,18 +345,18 @@ public void delete(Integer id) {
 
         try {
             session.beginTransaction();
-            Criteria criteriaProcessIds = session.createCriteria(Process.class).add(Restrictions.eq("processTemplate.processTemplateId", processTemplateId));
+            Criteria criteriaProcessIds = session.createCriteria(Process.class).add(Restrictions.eq(PARENTPROCESSTEMPLATEID, processTemplateId));
             criteriaProcessIds.setProjection(Projections.property("processId"));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "processId" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)